From 6d32d0a0ec7755c59d2da5c581f1d0dc4774d7e5 Mon Sep 17 00:00:00 2001
From: rvalue <i@rvalue.moe>
Date: Wed, 15 Dec 2021 07:02:09 +0000
Subject: [PATCH] Remove incorrect args for i18n.merge_file

The positional argument has been silently ignored for a time. And it causes failure
with "ERROR: Function does not take positional arguments" since meson 0.60.0
---
 data/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index 774618b3d..0077b49c5 100644
--- data/meson.build
+++ data/meson.build
@@ -4,7 +4,6 @@ install_man('gedit.1')

 appdata = 'org.gnome.gedit.appdata.xml'
 appdata_file = i18n.merge_file(
-  appdata,
   input: appdata + '.in',
   output: appdata,
   po_dir: '../po/',
@@ -23,7 +22,6 @@ endif

 desktop_file = 'org.gnome.gedit.desktop'
 desktop_output_file = i18n.merge_file(
-  desktop_file,
   type: 'desktop',
   input: desktop_file + '.in',
   output: desktop_file,
--
GitLab

